docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-product-offer-prices-feature-integration.md

```diff
@@ -92,8 +93,16 @@ Make sure that the following changes were applied in transfer objects:
 | PriceProductOfferCriteria | class | created | src/Generated/Shared/Transfer/PriceProductOfferCriteriaTransfer |
 | DataImporterReaderConfiguration | class | created | src/Generated/Shared/Transfer/DataImporterReaderConfigurationTransfer |
 | Currency.Code | attribute | created | src/Generated/Shared/Transfer/CurrencyTransfer |
-| PriceProductFilter.Filter | attribute | created | src/Generated/Shared/Transfer/PriceProductFilterTransfer |
+| MoneyValue | class | created |src/Generated/Shared/Transfer/MoneyValueTransfer |
+| PriceProduct | attribute | created | src/Generated/Shared/Transfer/PriceProductTransfer |
+| PriceType | class | created |src/Generated/Shared/Transfer/PriceTypeTransfer |
+| PriceProductFilter | class | created |src/Generated/Shared/Transfer/PriceProductFilterTransfer |
+| PriceProductFilter.quantity | class | created |src/Generated/Shared/Transfer/PriceProductFilterTransfer |
+| PriceProductFilter.filter | class | created | src/Generated/Shared/Transfer/PriceProductFilterTransfer |
 | PriceProductFilter.value | attribute | created | src/Generated/Shared/Transfer/PriceProductFilterTransfer |
+| PriceProductVolume | class | created |  src/Generated/Shared/Transfer/PriceProductVolumeTransfer |
+| PriceProductVolumeCollection | class | created |  src/Generated/Shared/Transfer/PriceProductVolumeCollectionTransfer |
+| PriceProductCriteria | class | created | src/Generated/Shared/Transfer/PriceProductCriteriaTransfer |
```

Comment:
please leave only transfers which appear with this feature and don't exist in Prices feature